### ISSUE TITLE
Cannot compile flipper contract

### ISSUE THREAD

**Issue by ayeteng** [paritytech/ink#391]
```
ubuntu@ubuntu1804:~/tmp/flipper$ cargo +nightly contract build
 [1/4] Collecting crate metadata
 [2/4] Building cargo project
   Compiling flipper v0.1.0 (/tmp/.cargo-contract_zD0iFQ)
error: failed to get bitcode from object file for LTO (Bitcode section not found in object file)

error: aborting due to previous error

error: could not compile `flipper`.

To learn more, run the command again with --verbose.
ERROR: xbuild failed with status exit code: 101
```

Version:
get flipper by cargo contract new flipper
cargo-contract v0.6.0
stable-x86_64-unknown-linux-gnu unchanged - rustc 1.43.1 (8d69840ab 2020-05-04)
nightly-x86_64-unknown-linux-gnu unchanged - rustc 1.45.0-nightly (9912925c2 2020-05-10)

is this relevant here?

**Comment by Robbepop** [comment]
What seems to be the source of the problem:
https://github.com/rust-osdev/cargo-xbuild/issues/69

**Comment by ascjones** [comment]
I think caused by https://github.com/rust-lang/cargo/pull/8192.

I'm currently investigating this. In the meantime it will work using an older nightly release e.g.

`cargo +nightly-2020-03-11 contract build`

Note that you may need to install `rust-src` for that with `rustup component add rust-src --toolchain nightly-2020-03-11`

**Comment by riusricardo** [comment]
Fixed on [#52](https://github.com/paritytech/cargo-contract/pull/52)
